packages/ui/primitives/document-flow/add-fields.tsx

```diff
@@ -225,6 +223,38 @@ export const AddFieldsFormPartial = ({
   const hasErrors =
     emptyCheckboxFields.length > 0 || emptyRadioFields.length > 0 || emptySelectFields.length > 0;
 
+  const fieldsWithError = useMemo(() => {
+    const fields = localFields.filter((field) => {
+      const hasError =
+        ((field.type === FieldType.CHECKBOX ||
+          field.type === FieldType.RADIO ||
+          field.type === FieldType.DROPDOWN) &&
+          field.fieldMeta === undefined) ||
+        (field.fieldMeta && 'values' in field.fieldMeta && field?.fieldMeta?.values?.length === 0);
+
+      return hasError;
+    });
+
```

Comment:
not sure about this. I remap it because the `FieldToolTip` expects a field of type `Field`.